### DISCUSSION TITLE
Alternator disconnect to prevent load dump

### DISCUSSION THREAD

**Discussion by seamaster101** [Louisvdw/dbus-serialbattery#436] [Q&A] [answered]
I have setup 4 X 300AH batteries with 4 X JK BMS. This battery system is located on a boat.
There is a major concern if at any point the battery BMSs disconnect due to high voltage or another reason while the alternator is running, there will over voltage of over 100V that is going to spike through the system (load dump) and possibly fry other equipment on the boat.
in this setup I'm using WS-500 smart alternator regulator that could stop the alternator charging when the BMS is about to disconnect due to hight voltage. Even if the WS-500 executes stops charging, the alternator require about 500mS to stop generating to prevent the load dump. The question is can JK BMS trigger alarm before any FET disconnect command is issued and is it possible to delay the FET disconnect let's say by 2sec just to be on the safe side?

**Comment by gmanic** [comment, marked as answer]
I'd suggest a load dump protection close to the alternator, like Sterling Power Protect.
I installed it in my mobile home, but have not (actively) tested it.

**Comment by mr-manuel** [comment]
The JKBMS unfortunately cannot do that.
